<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelDatasClass</class>
 <widget class="QWidget" name="PanelDatasClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_37" rowstretch="2,1">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_30">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Experience</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_38">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_21" stretch="1,0">
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_28" columnstretch="0,1,0,1">
            <item row="1" column="3">
             <widget class="QSpinBox" name="spinBoxClassInflation">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="minimum">
               <number>-999999999</number>
              </property>
              <property name="maximum">
               <number>999999999</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Base:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Initial level:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBoxClassInitialLevel">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="minimum">
               <number>-999999999</number>
              </property>
              <property name="maximum">
               <number>999999999</number>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Max level:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QSpinBox" name="spinBoxClassMaxLevel">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="minimum">
               <number>-999999999</number>
              </property>
              <property name="maximum">
               <number>999999999</number>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spinBoxClassBase">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="minimum">
               <number>-999999999</number>
              </property>
              <property name="maximum">
               <number>999999999</number>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Inflation:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonSetClassValues">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Set to class values</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_20">
       <property name="title">
        <string>Statistics progression</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_26">
        <item row="0" column="0">
         <widget class="WidgetSuperTree" name="treeViewStatisticsProgression">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="indentation">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_29">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Caracteristics</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_27">
        <item row="0" column="0">
         <widget class="QTreeView" name="treeView_2">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="groupBox_21">
       <property name="title">
        <string>Skills to learn</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_25">
        <item row="0" column="0">
         <widget class="WidgetSuperTree" name="treeViewClassSkills">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="indentation">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetSuperTree</class>
   <extends>QTreeView</extends>
   <header>widgetsupertree.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
